docs/code/conds/api/cron_kwargs.py

```diff
@@ -3,7 +3,7 @@
 @app.task(cron(minute="*/5"))
 def do_simple():
     "Run at every 5th minute"
-    ...
+
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).